src/go/api/scorch/app.go

```diff
@@ -326,6 +332,38 @@ func (this Scorch) stopFilebeat(ctx context.Context, cmd *exec.Cmd, port int) {
 	}
 }
 
+func (this Scorch) recordInfo(runID int, runDir string, name string, startTime time.Time, versionCommit string, versionTag string, versionDate string) error {
+	// runID, runDir, exp.Metadata.Name, start, version.Commit, version.Tag version.Date
+
+	var (
+		c    *mmcli.Command
+		resp string
+		err  error
+		info string
+	)
+
+	c = mmcli.NewCommand()
+	c.Command = "version"
+	if resp, err = mmcli.SingleResponse(mmcli.Run(c)); err != nil {
+		return fmt.Errorf("running minimega verssion %w", err)
+	}
+
+	info = fmt.Sprintf("Experiment Name: %s\nExperiment Commit Hash: pending\nScorch Run Name: %s\nStart Time: %v\nEnd Time: %v\nPhenix Version: %s %s %s \nMinimega Version: %s",
+		name, this.md.RunName(runID), startTime, time.Now().UTC(), versionCommit, versionTag, versionDate, resp)
+
+	if _, err = os.Stat(runDir); os.IsNotExist(err) {
```

Comment:
@nblair2 you don't really need to check that the directory exists before making it with `os.MkdirAll`, since `os.MkdirAll` will be a no-op if it already exists (no error will be returned).